Backend/Backend/Services/EpisodeService.cs

```diff
@@ -54,11 +84,28 @@ public class EpisodeService : IEpisodeService
 
 
         public async Task<bool> DeleteEpisode(Episode episode, DeleteEpisodeRequest deleteEpisodeRequest)
-        {  
-           await _fileService.DeleteFile(episode.AudioFileId.ToString()!);
-           _db.Remove(episode);
-           await _db.SaveChangesAsync();
-           return true;
+        {
+            Guid? guid = Guid.Parse(deleteEpisodeRequest.EpisodeId);
+            Files? file = await _db.File!.FirstOrDefaultAsync(u => u.FileId == episode.AudioFileId);
+
+            if (file != null)
+            {
+               var deleted = _fileService.DeleteFile(file.Path!);
```

Comment:
this is an optional change, but you dont have to  extract the deleted to a variable. you can just put the expression in the if statment.